api/src/main/java/io/opentelemetry/trace/EndSpanOptions.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opentelemetry.trace;
+
+import com.google.auto.value.AutoValue;
+import io.opentelemetry.common.Timestamp;
+import javax.annotation.Nullable;
+import javax.annotation.concurrent.Immutable;
+
+/**
+ * A class that enables overriding the default values used when ending a {@link Span}. Allows
+ * overriding the {@link Timestamp endTimestamp}.
+ *
+ * @since 0.1
+ */
+@Immutable
+@AutoValue
+public abstract class EndSpanOptions {
+  private static final EndSpanOptions DEFAULT = builder().build();
+
+  /**
+   * The default {@code EndSpanOptions}.
+   *
+   * @since 0.1
+   */
+  public static EndSpanOptions getDefault() {
```

Comment:
Not sure this is needed for the public API. Let's remove until someone asks for it.

Reply:
Works for me.